scss.scss

```diff
@@ -1,8 +1,6 @@
 @forward "src/scss";
 
 @use "./src/components/link";
-
```

Comment:
deliberate?

Reply:
yeah, seems that someone added space?  The template just amends the lines, so additional template additions are not double spaced.